assets/css/main.css

```diff
@@ -57,8 +57,17 @@ figcaption {
 }
 
 footer {
-    color: #F4F4F4 !important;
-    background-color: #095326 !important;
+    width: 100%;
+    padding-top: 1em;
+    padding-bottom: 1em;
+    color: #F4F4F4;
+    background-color: #095326;
+}
+
+footer>div {
+    max-width: 1200px;
```

Comment:
**question**: This value should be the same as:
```
.main-container>.main-item>div {
max-width: 1200px;
```
right? If so maybe it would be good to have some variable or something to handle that,

Reply:
Added CSS vars for this width and some colors; we can consider other vars in the next PRs.